<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конструктор визуальных новелл</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1281</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Добро пожаловать</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="projectsGrd">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <property name="spacing">
        <number>40</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QPushButton" name="deleteProject">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="playProject">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Играть</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editProject">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Редактировать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createProject">
         <property name="text">
          <string>Создать проект</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
